packages/contracts/contracts/ChainlinkAdapter.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {AggregatorV3Interface} from "./Dependencies/AggregatorV3Interface.sol";
+
+contract ChainlinkAdapter is AggregatorV3Interface {
+    uint8 public constant override decimals = 18;
+    uint256 public constant override version = 1;
+
+    /**
+     * @notice Maximum number of resulting and feed decimals
+     */
+    uint8 public constant MAX_DECIMALS = 18;
+    uint80 public constant CURRENT_ROUND = 2;
```

Comment:
not quite sure why this is hardcoded?

Reply:
decimals: optimistic assumption that it will never change.

current round is used by the adapter. it's not the real round. This is because it's combining multiple feeds into a "synthetic" feed.

so:
- current round = 2
- previous round = 1

Reply:
I might mistake the `roundId` with the `phaseId` as reported by [recent watchpug comment](https://notes.watchpug.com/p/18d3a3b1769SbE5X#n_2_h3_0). If the purpose of `adapter` is to mimic the interface of ChainLink design for `round`, I agree a hardcoded value should suffice.